.github/pull_request_template.md

```diff
@@ -1,7 +1,9 @@
 :warning: This application is Continuously Deployed: :warning:
 
-- Merged changes are automatically deployed to staging and production.
+- Application changes are automatically deployed to staging and production.
 
 - Make sure you follow [the guidance for deployments](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) **before** you merge.
 
 - Check your branch is being deployed in the [Release app](https://release.publishing.service.gov.uk/applications/smartanswers), after merging.
+
+- **Content** changes are **not** continuously deployed, and are [deployed using a different process](https://github.com/alphagov/smart-answers/blob/master/doc/smart-answer-flow-development/publishing.md).
```

Comment:
I think this only applies to start pages though

Reply:
I based this off the [message in the release app](https://release.publishing.service.gov.uk/applications/smartanswers) - I assume there are other content changes that it applies to? If it is just start pages then we should probably change it 👍

Reply:
i believe it is just the start pages 👍